### PR TITLE
Test node8 with config

### PR DESCRIPTION
**Pull request by ghost**
Changes include bumping the node version to v8 and the node-sass version to 4.5.3. The latter pointed up that the source repo being built from needed to be capable of being dynamically assigned, so I added a couple of build arguments to facilitate that. I also added a sample config file and made the docker-compose.yml notes an actual file. Last, I added build/run instructions to the README.

### CODE REVIEW

File: config/cm-libraries.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+xyzlib|http://simplye-dev-cm.amigos.org/xyzlib
```
Comment (jonathangreen) [inline config/cm-libraries.conf]: Was this file supposed to be include in the PR? I see it's also in the .gitignore, but it is referenced in the readme.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,71 @@
...
+    ```
+    git clone https://github.com/lyrasis/docker-patron-web.git ./patronweb
+    cd patronweb
+    ```
+
+3. As instructed in the README for the parent project, you must have already configured development access to the Circulation Manager from this local application by adding a sitewide setting to the Circulation Manager. Review that document as needed.
+
+4. Review the file `config/cm-libraries.conf.sample` for proper formatting in creating a configuration file which links the web application to a library's SimplyE Circulation Manager in order to pull catalog feeds into the display. The SimplyE web client application can be configured on behalf of multiple libraries to provide a web-based client for their patrons. In a multi-tenant environment, each library will be represented in the config file as a separate line.
+
+5. You can create an empty config file or rename the sample file with the filename `config/cm-libraries.conf`. Here's an example entry in the file for a library with a Circulation Manager short name of `abclib` whose Circulation Manager is available at the domain `circulation-simplye.library.org`:
```
Comment (jonathangreen) [inline README.md]: Is this supposed to be `config/cm-libraries.conf.sample`?

**Review by jonathangreen — COMMENTED** [review]
Had a minor comment about the `cm-libraries.conf` file. Other than that, this looks good to me.

**Comment by jonathangreen** [comment]
Should probably make a PR to https://github.com/NYPL-Simplified/circulation-patron-web to bump the node-sass version.

File: config/cm-libraries.conf (hunk 1 above)
Comment (ghost) [reply config/cm-libraries.conf]: No, it wasn't. I was playing around with possibilities and forgot the gitignore initially. Looks like this crept in before. It's not a big deal--it's just a demo system I've got up for development. But I should take it out before it's merged. I'll do that in a bit. But see comment below on what to include.

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: My thoughts aren't completely fleshed out on handling the config. In my other deployment repos, I've created samples for most other configs and not included the target file; there are either instructions for copying the sample to the primary file and editing (variables) or an Ansible template task that creates the appropriate file on the target from the variables files. For this particular file, the best way ultimately is to include the actual file with a commented example at the top. But I need to add a line to Amy's script that reads the file so it skips lines beginning with a #. I can make the change here to align with that goal. Does that sound good to you?

**Comment by ghost** [comment]
I've forked the main project and updated the node-sass version in a branch. If the above comment on the config file works for you, I'll also track down the one line I need to add to support comments in the config file and submit a pull request to the repo.

**Comment by jonathangreen** [comment]
I like the idea of having comments in the config file. I think probably dropping `cm-libraries.conf` from this PR makes lots of sense to me, and just having the sample file with some comments in it.

**Comment by ghost** [comment]
Will do. I'm adding code to skip comment lines, then I'll update my PR in a bit.

**Comment by ghost** [comment]
Okay, I think I have those changes committed, along with the --restart policy.

**Pull request merged**